.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Background
+
```

Comment:
It may be good to give some direction here as to what the correct and semantic title should be. See https://github.com/Uniswap/interface/blob/main/.github/pull_request_template.md?plain=1#L1-L17